docs/source/developer_tools.rst

```diff
@@ -45,6 +45,28 @@ Do not forget to cleanup after yourself:
     [host]$ docker-compose stop && docker-compose rm
 
 
+Make the container GPU-aware
+----------------------------
+
+Follow these instructions to launch containers leveraging the NVIDIA GPUs on the
+host machine:
+
+.. code:: bash
+
+    [host]$ cd docker/nvidia # go to the nvidia subdirectory
+    [host]$ ./setup_nvidia_docker_compose.py # extend the regular Compose file to leverage GPUs
+    [host]$ docker-compose build # add the CUDA development tools to the DTK base image
+    [host]$ docker-compose -p $USER up -d # as previously described
+
+Do not forget to set the environment for CUDA before you configure:
+
+.. code:: bash
+
+    [container]$ source ../scripts/docker_cuda_env.sh # set environment for CUDA
+    [container]$ ../scripts/docker_cmake # configure
+    [container]$ # now you may build and test
+
```

Comment:
We should make it more clear that the code above is executed inside of the `build` directory in the container per the previous docker instructions.